source/ConditionsStore.cpp

```diff
@@ -247,6 +248,14 @@ void ConditionsStore::Load(const DataNode &node)
 
 
 
+void ConditionsStore::Load(const DataFile &file)
+{
+	for(const DataNode &node : file)
```

Comment:
Maybe I'm missing something, but I get the impression from reading the code that each global conditions file gets saved with a `"conditions"` keyword, and then the actual conditions below the keyword.
So here I would expect a loop over the datafile to find the "conditions" keyword, and then an inner loop to load all conditions. (But I'm probably missing something.)

Reply:
Saving under a "global conditions" keyword is likely also more appropriate, to differentiate save game conditions with global conditions.

Reply:
Idk. Having a "global conditions" keyword instead of just using the existing "conditions" keyword, even though they're bothing loading to the same places, feels redundant to me with the file name. If the file name is already "global conditions," does the root node really also need to be "global conditions"? Can it not just be "conditions"?